sim_pipeline/gglens_table_test.py

```diff
@@ -0,0 +1,36 @@
+from astropy.table import Table

```

Comment:
this file should be in the tests/ folder as it is a test. Follow the conventions of the other test files